.moban.yaml

```diff
@@ -0,0 +1,15 @@
+overrides: coala.yaml
+
+language: ruby
+gitignore_language: Jekyll
+gitignore_extra_rulesets:
+  - GitBook
+
+configuration:
+  template_dir:
+    - ../coala-mobans/templates/
+    - ../coala-mobans/assets/
```

Comment:
`..`? What if I don't have coala-mobans lying around there?

Reply:
Then dont run `moban`.  :P  It does give a semi-decent error message.

fyi this is what we have done for coala & coala-bears.  .travis.yml clones https://gitlab.com/coala/mobans to that location.

Medium term goal is for moban to manage these dependencies https://github.com/moremoban/moban/issues/35 .  The logic has mostly been written in `yehua` (https://github.com/moremoban/yehua/blob/master/yehua/project.py#L29)